lib/reporters/spec.js

```diff
@@ -45,46 +45,60 @@ function Spec(runner, options) {
     return Array(indents).join('  ');
   }
 
-  runner.on(EVENT_RUN_BEGIN, function () {
+  runner.on(EVENT_RUN_BEGIN, function() {
     Base.consoleLog();
   });
 
-  runner.on(EVENT_SUITE_BEGIN, function (suite) {
+  runner.on(EVENT_SUITE_BEGIN, function(suite) {
     ++indents;
     Base.consoleLog(color('suite', '%s%s'), indent(), suite.title);
   });
 
-  runner.on(EVENT_SUITE_END, function () {
+  runner.on(EVENT_SUITE_END, function() {
     --indents;
     if (indents === 1) {
       Base.consoleLog();
     }
   });
 
-  runner.on(EVENT_TEST_PENDING, function (test) {
+  runner.on(EVENT_TEST_PENDING, function(test) {
     var fmt = indent() + color('pending', '  - %s');
     Base.consoleLog(fmt, test.title);
   });
 
-  runner.on(EVENT_TEST_PASS, function (test) {
+  runner.on(EVENT_TEST_PASS, function(test) {
+    function logRetries() {
+      if (test._currentRetry > 0) {
+        /*
+          Included message within format so it can be passed into `Base.consoleLog` as one argument
+        */
+        var retryFmt = color(
+          'bright yellow',
+          `  (retry x${test._currentRetry})`
```

Comment:
[Question] Is it intentional that this has two spaces in front of it, not one?

Reply:
It was a stylistic choice I made. However, one space seems to look better. Will change.